src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -1,32 +1,37 @@
 package mate.academy.hibernate.relations;
 
 import java.util.List;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
```

Comment:
Ensure that `CountryDaoImpl` is correctly implemented and that it handles session management properly.